src/site/includes/related-information.drupal.liquid

```diff
@@ -10,7 +10,7 @@
       <li>
         <p class="vads-u-margin--0">
           <strong>
-            <a href="{{ fieldRelatedInformationLink.entity.fieldLink.url.path }}">
+            <a onclick="recordEvent({ 'event': 'nav-linkslist', 'links-list-header': '{{ fieldRelatedInformationLink.entity.fieldLink.title }}', 'links-list-section-header': 'Related information' })" href="{{ fieldRelatedInformationLink.entity.fieldLink.url.path }}">
```

Comment:
Do we need to escape `fieldRelatedInformationLink.entity.fieldLink.title` or anything?

Reply:
Interesting, you mean for quote characters? I don't think so since they are usually just text... oh apostrophes perhaps, how would you suggest doing this?

Reply:
I think there is a humble ` | escape ` filter

Reply:
oh excellent! thanks!! 🙂